src/main/scala/temple/builder/project/ProjectConfig.scala

```diff
@@ -12,6 +12,7 @@ object ProjectConfig {
   val defaultLanguage: ServiceLanguage = ServiceLanguage.Go
   val defaultDatabase: Database        = Database.Postgres
   val defaultAuth: ServiceAuth         = ServiceAuth.Email
+  val authPort: Int                    = 1023 //* One before the [[Templefile]] start point
```

Comment:
This looks dangerous: 1023 needs sudo, right?

Reply:
Also wtf is that comment symbol?

Reply:
That is correct... I will update accordingly